### PR TITLE
p9: mac and windows tests

### PR DESCRIPTION
**Pull request by sevki**
closes: #14 

Signed-off-by: sevki <s@sevki.org>

### CODE REVIEW

File: .github/workflows/p9.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      run: |
+        go get -v -t -d ./...
+        if [ -f Gopkg.toml ]; then
+            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+            dep ensure
+        fi
+    - name: Build
+      run: go build -v ./...
+
+    - name: Test
```
Comment (hugelgupf) [inline .github/workflows/p9.yml:37]: is there a way to run this before we merge it?

i'm curious about the qemu-based integration test. may want to tag that as linux only.

https://github.com/hugelgupf/p9/blob/main/fsimpl/localfs/localfs_integration_test.go

File: .github/workflows/p9.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Get dependencies
+      run: |
+        go get -v -t -d ./...
+        if [ -f Gopkg.toml ]; then
+            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+            dep ensure
```
Comment (hugelgupf) [inline .github/workflows/p9.yml:32]: no need for these 4 lines, p9 is fully on go mod.

File: .github/workflows/p9.yml (hunk 1 above)
Comment (sevki) [reply .github/workflows/p9.yml:37]: oh yeah we should. let me give it a go :)

Comment (sevki) [reply .github/workflows/p9.yml:30]: Yeah the tests are definitely running on all PRs. and failing for both windows and mac. 
I think those should be seperate PRs to fix them tho.

Comment (sevki) [reply .github/workflows/p9.yml:30]: https://github.com/sevki/p9/runs/895589927?check_suite_focus=true

**Comment by sevki** [comment]
@hugelgupf PTAL

**Pull request merged**